services/hanke-service/src/main/resources/db/changelog/changesets/063-drop-haitta-columns-from-hanke.yml

```diff
@@ -0,0 +1,39 @@
+databaseChangeLog:
+  - changeSet:
+      id: 063-drop-haitta-columns-from-hanke
+      author: Teemu Hiltunen
+      comment: Drop haitta columns from hanke
+      preConditions:
+        - onFail: MARK_RAN
+          and:
+            - columnExists:
+                tableName: hanke
+                columnName: kaistahaitta
+            - columnExists:
+                tableName: hanke
+                columnName: kaistapituushaitta
+            - columnExists:
+                tableName: hanke
+                columnName: meluhaitta
+            - columnExists:
+                tableName: hanke
+                columnName: polyhaitta
+            - columnExists:
+                tableName: hanke
+                columnName: tarinahaitta
```

Comment:
We know these exist, so preconditions are not necessary.

Reply:
Yes, you are right. I had to add these when I jumped around with different haittaindeksi calculation branches and my database got changes from here and there. I forgot to delete these afterwards.

Reply:
Sometimes it would be nice to be able to roll back the schema changes. Too bad the two biggest migration managers in the Spring ecosystem don't support them.